ckanext/geocat/tests/test_distribution_metadata.py

```diff
@@ -82,9 +79,6 @@ def test_fields_values(self):
 
         self.assertIsNotNone(download)
 
-        # identifier
-        self.assertEquals('', download.get('identifier'))
```

Comment:
Have we stopped mapping all of these fields? I think at least `identifier` and `coverage` are still mapped, right? Let's keep tests for those at least, if they are.

Reply:
No they were never mapped for geocat. So these tests would fail if I still I keep them there. Really I think this is fine as it is now.